src/develop/ui/patterns/mobile/rangeslider.md

```diff
@@ -3,74 +3,74 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# RangeSlider Pattern
+# Range Slider 
 
-The RangeSlider pattern allows you to set a value by dragging a handle within a configured range. It is used to control a variable value with simple and interactive user input.
+You can use the Range Slider UI Pattern to allow users select a single value between two range values. This pattern enables the adjustment of content within a predetermined range. Moving the slider along the track, increases or decreases the value.  
 
-## How to Use the RangeSlider Pattern
+![](images/rangeslider-preview.png)
 
-Bind your variable to the **InitialValue** input and use the **OnChange** event to add your logic to handle value changes.
+## How to use the Range Slider UI Pattern
 
-![](images/range_slider.png)
+1. In Service Studio, in the Toolbox, search for  `Range Slider`. 
 
-1\. After setting the **MinValue** , **MaxValue** and the **InitialValue** , you'll need to create the **OnChange** event.
+    The Range Slider widget is displayed.
 
-![](images/range_slider_on_change.png)
+    ![](images/rangeslider-widget.png)
 
-2\. Create an integer value and assign it.
+1. From the Toolbox, drag the Range Slider widget onto your application's screen.
+1. Bind your variable to the **InitialValue** input and use the **OnChange** event to add your logic to handle value changes.
```

Comment:
We don't seem to be binding a variable anywhere, nor is there an "InitialValue" input... Do we mean property?